jerry-libc/jerry-libc-printf.c

```diff
@@ -252,7 +253,7 @@ libc_printf_write_d_i (FILE *stream, /**< stream pointer */
 
     case LIBC_PRINTF_ARG_LENGTH_TYPE_HIGHL:
     {
-      LIBC_UNREACHABLE ();
+      assert (!"unsupported length field L");
```

Comment:
Is this ! is a widely used form? It looks strange to me, although I understand its purpose.